test/org/loklak/harvester/TwitterScraperTest.java

```diff
@@ -43,8 +39,8 @@ public void test_prepareSearchURL() {
     }
 
     @Test
-    public void test_search() {
```

Comment:
Is this really needed by JUnit? If not, please make it into a separate commit because this is completely unrelated to this change.